extensions/binding/org.eclipse.smarthome.binding.weatherunderground/README.md

```diff
@@ -131,7 +131,7 @@ demo.items:
 String Conditions "Conditions [%s]" {channel="weatherunderground:weather:CDG:current#conditions"}
 Image Icon "Icon" {channel="weatherunderground:weather:CDG:current#icon"}
 DateTime ObservationTime "Observation time [%1$tH:%1$tM]" <clock>  {channel="weatherunderground:weather:CDG:current#observationTime"}
-String Location "Location [%s]" {channel="weatherunderground:weather:CDG:current#location"}
+String ObservationLocation "Location [%s]" {channel="weatherunderground:weather:CDG:current#location"}
```

Comment:
Location of xeather observation.
It is the terminology of WU: https://www.wunderground.com/weather/api/d/docs?d=data/conditions